client/stylesheets/_toCat.scss

```diff
@@ -4603,8 +4610,8 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .searchRefinePrice li input {
-    width: 30px;
-    margin-left: 2px;
+    width: 45px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, margin-left, width